karapace/schema_models.py

```diff
@@ -180,6 +183,50 @@ def schema(self) -> Draft7Validator | AvroSchema | ProtobufSchema:
         return parsed_typed_schema.schema
 
 
+class AvroMerge:
+    def __init__(self, schema_str: str, dependencies: Mapping[str, Dependency] | None = None):
+        self.schema_str = json_encode(json_decode(schema_str), compact=True, sort_keys=True)
+        self.dependencies = dependencies
+        self.unique_id = 0
```

Comment:
Could this be a random UUID?

Reply:
When we have two schemas which we must compare, these schemas' IDs must be the same. So, in our way, it works because schema numbers start from zero. For a random ID solution, we must somehow start numbering the two comparable schemas with the same 'random' ID. This is possible, but instead of using random values, we must use a hash of the current subject, which will be the same for both schemas in comparison. We must change the interface of some functions to provide the subject variable to the parser of schemas.It is runtime data which is never stored in Kafka, so is there a sense in using UUID?

Reply:
Ok, Makes sense.